modular_skyrat/modules/knives/knives.dm

```diff
@@ -0,0 +1,40 @@
+//--- BOWIE'S KNIFE (bowie knife)---\\\
+
+
+/obj/item/melee/knife/bowie
+	name = "Bowie Knife"
+	desc = "A frontierman's classic, closer to a shortsword then a knife, with a full-tanged build, brass handguard and pommel, a wicked sharp point, and a large, heavy blade, it's almost everything you could want in a knife, besides portability."
+	icon = 'modular_skyrat/modules/knives/icons/bowie.dmi'
+  	iconstate = 'bowie'
+	lefthand_file = 'modular_skyrat/modules/knives/icons/bowie_lefthand.dmi'
+	righthand_file = 'modular_skyrat/modules/knives/icons/bowie_righthand.dmi'
+  	worn_icon_state = "knife"
+	force = 20 // Zoowee Momma!
+	w_class = WEIGHT_CLASS_NORMAL
+	throwforce = 15
+	wound_bonus = -15
+	bare_wound_bonus = 20
+
+  
+  /obj/item/storage/bag/bowiesheathe
+	name = "Bowie Knife sheathe"
+	desc = "A dressed up leather sheathe, featuring a bass tip, and a large pocket clip right in the center, for ease of carrying an otherwise burdersome knife."
```

Comment:
```suggestion
	desc = "A dressed-up leather sheath featuring a brass tip. It has a large pocket clip right in the center, for ease of carrying an otherwise burdensome knife."
```